_site/index.html

```diff
@@ -739,6 +739,33 @@ <h3>T-Z</h3>
         </div>
         </section>
 
+        <section class="block__word list">
+          <p class="word__signal"></p>
+          <p class="word__title" id="polyamory">
+            Polyamory
+            <span class="word__speech">
+              noun
+            </span>
+          </p>
+
+          <p class="word__definition">
+            belief that romantic and/or sexual exclusivity is not necessary for a happy, committed relationship;
+            an umbrella term for various types of relationships where all partners involved consent to non-monogamy,
+            such as open relationships, polycules, throuples, group marriages, etc.
+          </p>
+
+          <div class="block__type">
+            <p class="word__type">
+              Note
+            </p>
+            <p class="word__breakdown">
+              <span class="word__alt">
+                One can be polyamorous despite currently being single or in a monogamous relationship.
+              </span>
+            </p>
```

Comment:
Is there a reason you excluded the resource you shared in the comment from a `Further Reading` section? If not, I think any links are always helpful for people to read more.

Reply:
Because I only thought about writing this blog post a day after I have submitted the PR 😅
If you think it's the right place for this link, I'll add it :)

Reply:
🙈😅 That makes sense! I'd love to include your blog post (as I thought it was very personal and helpful to me, and I appreciated the shoutout to the dictionary).

Maybe you could add 1 or 2 external ones that you like to help round out the perspective? (I try to include a few per word to illustrate the richness of words.) I trust your suggestions.